### DISCUSSION TITLE
Confidentiality of data

### DISCUSSION THREAD

**Discussion by marcoscale98** [obiscr/ChatGPT#320] [Q&A]
Hi all,

does the plugin developed in this repository save or might save in the future the data from requests to OpenAI or the codebase of the project I am working on?

Thanks

**Comment by obiscr** [comment]
Hi, your source code is never stored on our servers and never used for training. We use embeddings to provide codebase aware answers. For details with regards to the data we collect & store, please refer to our [data collection & privacy policy](https://www.easycode.ai/privacy).
